index.d.ts

```diff
@@ -1,3 +1,5 @@
+import Ajv from "ajv";
```

Comment:
Could TS users that add ajv7 instance have trouble with this interface that it is ajv6?

Reply:
Yeah they're going to have trouble, ajv 8 removes some properties from the options object and adds new ones the options object type between version 6, 7 and 8 are similar but is not compatible I'm not entirely sure on how to deal with this issue.